x/escrow/handler_test.go

```diff
@@ -725,23 +713,29 @@ func (q query) check(t testing.TB, db weave.ReadOnlyKVStore, qr weave.QueryRoute
 	t.Helper()
 
 	h := qr.Handler(q.path)
-	require.NotNil(t, h, q.path)
+	assert.Equal(t, true, h != nil)
```

Comment:
I think this is hard to read. The use of `assert.Equal` seems so forced. 

I think that those two lines (old version and the new one) are not testing the same thing.

Reply:
they are, the second one does not provide a message.

It indeed feels pretty forced and I can't help but wonder why we don't have a helper for that. 

It does not seem right to go with
```
if err != nil {
   t.Fatal...
}
```
Because it's super verbose compared to the alternatives. These refactorings come from a discussion with @ethanfrey sometime ago, where I raised a concern about us not having helpers for anything, except `Equal` and `Nil`, I remember the answer was that we can actually use `Equal` for `NotNil`. 

While we can it's indeed not very informative. But better than a 3 liner.